spec/core/block/schemas/ImportanceBlockDTO.yml

```diff
@@ -0,0 +1,23 @@
+type: object
+allOf:
+  - $ref: "./BlockDTO.yml"
+  - type: object
+    required:
+      - votingEligibleAccountsCount
+      - harvestingEligibleAccountsCount
+      - totalVotingBalance
+      - previousImportanceBlockHash
+    properties:
+      votingEligibleAccountsCount:
+        $ref: ../../../schemas/UInt32.yml
+        description: Number of voting eligible accounts.
+      harvestingEligibleAccountsCount:
+        type: string # uint64
```

Comment:
can this be a reference to one of the schemas? Maybe create a UInt64.yml if not using Amount.yml?
This is for the java generation that automatically converts strings to BigInteger

Reply:
@rg911 , pushed this change. PR approved